src/tests/GC/Features/BackgroundGC/foregroundgc.cs

```diff
@@ -12,6 +12,7 @@ namespace ForegroundGC
     class ForegroundGC
     {
         static bool done = false;
+        readonly static object _lock = new object();
```

Comment:
Can we use the new `System.Threading.Lock` instead?

Reply:
Let's keep things super simple for now. This is something we will consider after the first wave of testing.